localops/src/process.rs

```diff
@@ -0,0 +1,66 @@
+use std::path::PathBuf;
+use std::process::{Command, Stdio};
+
+use anyhow::{anyhow, Result};
+#[cfg(any(target_os = "macos", target_os = "linux"))]
+use nix::sys::signal::{kill, Signal};
+#[cfg(any(target_os = "macos", target_os = "linux"))]
+use nix::unistd::Pid;
+
+pub fn daemonize(bin_path: PathBuf) -> Result<()> {
+    daemonize_process(bin_path)
+}
+
+#[cfg(not(any(target_os = "macos", target_os = "linux", target_os = "windows")))]
+fn daemonize_process(bin_path: PathBuf) -> Result<()> {
+    Err(anyhow!(
+        "deamonizing processes is not supported on your operating system"
+    ))
+}
+
+#[cfg(any(target_os = "macos", target_os = "linux"))]
+fn daemonize_process(bin_path: PathBuf) -> Result<()> {
+    // ¯\_(ツ)_/¯
+    let status = Command::new("bash")
+        .arg("-c")
+        // TODO(b5): might be nice to capture output in a log file at some point?
```

Comment:
Yes, please make an issue, this is unusable to debug anything atm, so this should take some config of where to put the log file

Reply:
how about we default to `$daemon-name.log` in the application cache directory for now?

Reply:
aren‘t there any defaults for log files?

Reply:
this PR now writes to log files